fluss-flink/fluss-flink-common/src/test/java/com/alibaba/fluss/flink/source/deserializer/FlussDeserializationSchemaTest.java

```diff
@@ -0,0 +1,167 @@
+        OrderDeserializationSchema schema = new OrderDeserializationSchema();
+
+        Order result = schema.deserialize(scanRecord);
+
+        assertThat(result.getOrderId()).isEqualTo(1002L);
+        assertThat(result.getItemId()).isEqualTo(5002L);
+        assertThat(result.getAmount()).isEqualTo(4);
+        assertThat(result.getAddress()).isEqualTo("456 Oak Ave");
+    }
+
+    @Test
+    public void testDeserializeWithNullValues() throws Exception {
+        GenericRow row = new GenericRow(4);
+        row.setField(0, 1003L);
+        row.setField(1, 5003L);
+        row.setField(2, 5);
+        row.setField(3, null);
+
+        ScanRecord scanRecord = new ScanRecord(row);
+        OrderDeserializationSchema schema = new OrderDeserializationSchema();
+
+        Order result = schema.deserialize(scanRecord);
+
+        assertThat(result.getOrderId()).isEqualTo(1003L);
+        assertThat(result.getItemId()).isEqualTo(5003L);
+        assertThat(result.getAmount()).isEqualTo(5);
+        assertThat(result.getAddress()).isEqualTo("null");
+    }
+
+    @Test
+    public void testDeserializeWithNullRecord() {
+        OrderDeserializationSchema schema = new OrderDeserializationSchema();
+
+        assertThatThrownBy(() -> schema.deserialize(null)).isInstanceOf(NullPointerException.class);
+    }
+
+    @Test
+    public void testGetProducedType() {
+        OrderDeserializationSchema schema = new OrderDeserializationSchema();
+        TypeInformation<Order> typeInfo = schema.getProducedType();
+
+        assertThat(typeInfo).isNotNull();
+        assertThat(typeInfo.getTypeClass()).isEqualTo(Order.class);
+    }
+
+    @Test
+    public void testJsonStringDeserialize() throws Exception {
+        // Create test data
+        GenericRow row = new GenericRow(2);
+        row.setField(0, 42L);
+        row.setField(1, BinaryString.fromString("test value"));
+        ScanRecord scanRecord = new ScanRecord(row);
+
+        // Create deserializer
+        JsonStringDeserializationSchema deserializer = new JsonStringDeserializationSchema();
+        // Test deserialization
+        deserializer.open(null);
+        String result = deserializer.deserialize(scanRecord);
+
+        // Verify result
+        assertThat(result).isNotNull();
+        assertThat(result)
+                .isEqualTo(
+                        "{\"row\":\"(42,test value)\",\"offset\":-1,\"timestamp\":-1,\"changeType\":\"INSERT\"}");
+    }
+
+    @Test
+    public void testStringGetProducedType() {
+        // Create deserializer
+        JsonStringDeserializationSchema deserializer = new JsonStringDeserializationSchema();
+
+        // Get type information
+        TypeInformation<String> typeInfo = deserializer.getProducedType();
+
+        // Verify type information
+        assertThat(typeInfo).isNotNull();
+        assertThat(typeInfo.getTypeClass()).isEqualTo(String.class);
+    }
+
+    @Test
+    public void testSerializable() throws Exception {
```

Comment:
This seems just test a testing class (`OrderDeserializationSchema`) which is not necessary IMO.